packages/react-dom/src/events/ReactBrowserEventEmitter.js

```diff
@@ -125,62 +150,74 @@ function getListeningForDocument(mountAt: any) {
  */
 export function listenTo(
   registrationName: string,
-  mountAt: Document | Element,
-) {
-  const isListening = getListeningForDocument(mountAt);
+  mountAt: Document | Element | Node,
+  isLegacy: boolean,
+): void {
```

Comment:
This changes behavior from `isLegacy` effectively being `true` (because it didn't exist) to `false` by default. Are we sure we have no internal callsites calling this? Are they unaffected?

Reply:
I’ve check all callsites and updated them